OrbitGl/ModulesDataView.cpp

```diff
@@ -105,31 +104,26 @@ void ModulesDataView::DoSort() {
 const std::string ModulesDataView::MENU_ACTION_MODULES_LOAD = "Load Symbols";
 const std::string ModulesDataView::MENU_ACTION_MODULES_VERIFY =
     "Verify Frame Pointers";
-const std::string ModulesDataView::MENU_ACTION_DLL_FIND_PDB = "Find Pdb";
-const std::string ModulesDataView::MENU_ACTION_DLL_EXPORTS = "Load Symbols";
 
 //-----------------------------------------------------------------------------
 std::vector<std::string> ModulesDataView::GetContextMenu(
-    int a_ClickedIndex, const std::vector<int>& a_SelectedIndices) {
+    int clicked_index, const std::vector<int>& selected_indices) {
   bool enable_load = false;
-  bool enable_dll = false;
   bool enable_verify = false;
-  if (a_SelectedIndices.size() == 1) {
-    std::shared_ptr<Module> module = GetModule(a_SelectedIndices[0]);
-    if (!module->GetLoaded()) {
-      if (module->m_FoundPdb) {
+  if (selected_indices.size() == 1) {
+    std::shared_ptr<Module> module = GetModule(selected_indices[0]);
+    if (!module->IsLoaded()) {
```

Comment:
The if-else here is a bit confusing. What about
```
if (module->IsLoadable() && !module->IsLoaded()) {
  enable_load = true;
}
if (module->IsLoaded()) {
  enable_verify = true;
}
```
?

Reply:
done - also removed top-level if - there is no need to check size of selected indices.